### ISSUE TITLE
Show site stats to users in dashboard

### ISSUE THREAD

**Issue by ottok** [Seravo/seravo-plugin#5]
As part of the WP-palvelu core experience, customers should see access log stats in their dashboard to know what service plan they should have for their traffic amount.

Consider including the functionality of https://github.com/Seravo/wp-dashboard-request-stats into this plugin.

**Comment by ottok** [comment]
Stats are now visible under Tools > Reports, but an additional dashboard widget would be cool.

**Comment by l3ku** [comment]
@nasusa please design a new dashboard widget that will replace the current "Error log count" widget.

**Comment by ottok** [comment]
Dashboard widget (=on WP-admin front page) would still be a nice thing.

**Comment by Moppa5** [comment]
I sketched how the dashboard widget could look like but I'm unsure whether the widget design should be different. Couldn't we use the Site Status page HTTP Request layout for this, or should the dashboard have something else? Also, I added to the sketch some colors to indicate if there are more requests than the plan allows. This new "Site Status" dashboard widget could have the basic PHP error count notification as well. In addition, some test results like feature #66 will provide, could be added to this widget, and they could replace the mere PHP error count message. 

![site_statistics](https://user-images.githubusercontent.com/16706187/111786290-4e178b00-88c6-11eb-9485-6837e08beeff.png)
